pkg/agent/multicast/mcast_controller.go

```diff
@@ -264,9 +264,16 @@ type Controller struct {
 	queryGroupId binding.GroupIDType
 	// nodeGroupID is the OpenFlow group ID in OVS which is used to send IGMP report messages to other Nodes.
 	nodeGroupID binding.GroupIDType
+	// forwardInboundGroupID is the OpenFlow group ID in OVS which is used to forward inbound multicast traffic
+	// from uplink when flexibleIPAM is enabled.
+	forwardInboundGroupID binding.GroupIDType
+	// forwardOutboundGroupID is the OpenFlow group ID in OVS which is used to forward outbound multicast traffic
+	// from host interface when flexibleIPAM is enabled.
+	forwardOutboundGroupID binding.GroupIDType
```

Comment:
Can we merge thet two groups ( "forwardOutboundGroupID" and "forwardInboundGroupID" ) into one? OVS datapath can ensure it would not output a packet to the port where it enters OVS pipeline when using "output:xxx" action.

Reply:
I am thinking is it easier if we process the hostIface port the same as uplink (add it as a default port for any group), then we don't need to introduce such additional groups, instead, we can add flows to ensure packets entering OVS from hostInterface and uplink to be resubmitted to MulticastRoutingTable (skip egress rules)?

Reply:
`forwardOutboundGroupID` has been deleted wenyingd suggested. In addition, the host interface and the uplink interface are processed in the same way. They are both resubmitted to MulticastRoutingTable when packets enter OVS, and both get outputted.